README.md

```diff
@@ -0,0 +1,33 @@
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+ 
+- [👥 Authors](#authors)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Add-Linters-Project] <a name="about-project"></a>
+
+
+
+**[Add-Linters-Project]** is a project to demonstrate ability of adding linters to
+project by saying hello to microverse.
+
+
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Author**
+
+- GitHub: [@zunairkhan811](https://github.com/zunairkhan811)
```

Comment:
- [ ] Kindly I recommend you to update your README file, README file should be descriptive and **All main headers** from the [README template](https://github.com/microverseinc/readme-template) should be present in a yours repo and Please Make sure the you has added the **section Future Features**, including 1-3 potential features to be added..